src/core/reopt.jl

```diff
@@ -144,7 +144,7 @@ function run_reopt(ms::AbstractArray{T, 1}, p::REoptInputs) where T <: JuMP.Abst
 		Threads.@threads for i = 1:2
 			rs[i] = run_reopt(inputs[i])
 		end
-		if typeof(rs[1]) <: Dict && typeof(rs[2]) <: Dict
+		if typeof(rs[1]) <: Dict && typeof(rs[2]) <: Dict && rs[1]["status"] != "error" && rs[2]["status"] != "error"
```

Comment:
Does this fix the TODO comment below?

Reply:
Depends on what the anticipated functionality was in the TODO. Right now, if rs[1] is optimal but rs[2] errors out, then both rs[1] and rs[2] are returned where rs[2] will contain the true error. we can add a @warn before returning `rs` to alert the user that they will receive a vector of Dict elements.

Reply:
We'd have to handle infeasible models on line 529. instead of returning `m` we might want to return a simple dictionary with termination status? maybe invoke the error handler here.

Reply:
Not trying to add work here, just wondering what the change was. Feel free to proceed as you'd like even if it doesn't address the TODO.

Reply:
We might have resolved the TODO to some extent... we throw an error if models are infeasible, which returns a dictionary that contains model infeasibility warning as well as an error text.